Makefile

```diff
@@ -3,12 +3,35 @@ SUB_PROJECTS := sdk/dotnet sdk/nodejs sdk/python sdk/go
 include build/common.mk
 
 PROJECT         := github.com/pulumi/pulumi
-PROJECT_PKGS    := $(shell go list ./cmd/... ./pkg/... | grep -v /vendor/)
+PROJECT_PKGS    := $(shell go list ./pkg/... | grep -v /vendor/)
 EXTRA_TEST_PKGS := $(shell go list ./examples/ ./tests/... | grep -v tests/templates | grep -v /vendor/)
 VERSION         := $(shell scripts/get-version HEAD)
 
 TESTPARALLELISM := 10
 
+ensure::
+	$(call STEP_MESSAGE)
+ifeq ($(NOPROXY), true)
+	@echo "cd sdk && GO111MODULE=on go mod tidy"; cd sdk && GO111MODULE=on go mod tidy
+	@echo "cd sdk && GO111MODULE=on go mod vendor"; cd sdk && GO111MODULE=on go mod vendor
```

Comment:
I thought we were switching to `go mod download`?

Reply:
I tried this with `go mod download` and got errors. I'll give this another try and open up a follow up PR.